tests/Lykke.Job.CandlesHistoryWriter.Tests/CandlesCacheInitializationTest.cs

```diff
@@ -106,15 +119,15 @@ public void InitializeTest()
                                     It.Is<CandleTimeInterval>(i => i == interval),
                                     It.Is<CandlePriceType>(p => p == priceType),
                                     It.Is<DateTime>(d => d == now.TruncateTo(interval).AddIntervalTicks(-AmountOfCandlesToStore, interval)),
```

Comment:
The parameter 1 in AddIntervalTicks is calculated by a different way comparing with CacheAssetPairCandlesAsync method above.

Reply:
Yes, way is different, but result is the same. Because of different base value for calculation. Here ```now``` is used for both ```to``` and ```from```